main/src/library/Foldable.flix

```diff
@@ -388,4 +388,9 @@ mod Foldable {
     pub def fold(t: t[a]): a \ Foldable.Aef[t] with Foldable[t], Monoid[a] =
         Foldable.foldLeft((acc, x) -> Monoid.combine(acc, x), Monoid.empty(), t)
 
+    ///
+    /// Returns the length of `t`, i.e. the number of times it can be folded over.
```

Comment:
Its not the number of times it can be folded over? Its the number of elements.

Reply:
It is copied from `Foldable.length`